src/v/utils/base64.cc

```diff
@@ -122,24 +114,10 @@ ss::sstring iobuf_to_base64(const iobuf& input) {
 }
 
 bytes base64url_to_bytes(std::string_view data) {
-    bytes rv{bytes::initialized_later{}, data.size()};
-    unsigned int bits_collected = 0;
-    unsigned int accumulator = 0;
-    int pos = 0;
-    for (const unsigned char c : data) {
-        if (c > 127 || decode_url_table[c] < 0) {
-            throw base64_url_decoder_exception();
-        }
-        accumulator = (accumulator << 6)
-                      + static_cast<unsigned int>(decode_url_table[c]);
-        bits_collected += 6;
-        if (bits_collected >= 8) {
-            auto val = static_cast<bytes::value_type>(
-              (accumulator >> (bits_collected - 8)) & 0xffu);
-            rv[pos++] = val;
-            bits_collected -= 8;
-        }
+    std::string srv;
+    if (!absl::WebSafeBase64Unescape(data, &srv)) {
+        throw base64_url_decoder_exception{};
     }
-    rv.resize(pos);
-    return rv;
+    // NOLINTNEXTLINE
```

Comment:
nit: could be a bit more specific

```suggestion
    // NOLINTNEXTLINE (cppcoreguidelines-pro-type-reinterpret-cast)
```
I think that's the right incantation anyway

Reply:
always a good idea to be specific on these nolints